src/locales/en/common.json

```
@@ -593,7 +594,10 @@
     },
     "requests": {
       "messageRequest": "Signature request",
-      "transactionRequest": "Transaction request",
+      "transactionRequest": {
+        "title": "{{app}} on {{chain}}",
+        "transactionRequest":"Transaction request"
+      },
```

Used key nesting looked a bit wierd, maybe we could use a flat structure as below:

```suggestion
      "transactionRequest": "Transaction request".
      "transactionRequestFormat": "{{app}} on {{chain}}",
```